### DISCUSSION TITLE
Google group replacement

### DISCUSSION THREAD

**Discussion by buchdag** [nginx-proxy/nginx-proxy#1623] [General]
Hi folks,

This is somewhat parallel to #1304. As you probably all noticed the historic Google group has been defunct for a while now.

I was thinking of the possible replacements if there is a strong enough demand for it from the user's community.

The possible platforms I had in mind are:
- GitHub Discussions
- Discord
- Gitter maybe ?

This is open to suggestions and discussions 👍

**Comment by tkw1536** [comment]
I personally think GitHub discussions are a good option, so that we don't have to fracture the community across different platforms.

**Comment by buchdag** [comment]
> I personally think GitHub discussions are a good option, so that we don't have to fracture the community across different platforms.

Agreed, and the ability to transfer an issue to discussions sounds pretty nice.

**Comment by buchdag** [comment]
Let's give GitHub Discussions a try then.
